decidim/decidim
# Fix notifications page when vapid is not available

#### :tophat: What? Why?
When Decidim is being upgraded to version 0.27, an error is being generated on the user notifications page by the feature added via #8774. The notifications page is looking for vapid related secrets that may not be available. 

This PR adds a fix in handling the vapid related keys.

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #10284

#### Testing
1. To simulate the upgrade from previous versions comment in config/secrets.yml the vapid related keys 
2. Restart the application
3. Login as admin and visit Notifications Settings page ( in My account menu)
4. You can see the error 
5. Apply this Patch 
6. Refresh the page. (Now there should not be any VAPID related errors )

### :camera: Screenshots
*Please add screenshots of the changes you're proposing*
![Description](URL)

:hearts: Thank you!
